dev/breeze/src/airflow_breeze/prepare_providers/provider_documentation.py

```diff
@@ -1222,6 +1222,13 @@ def _regenerate_pyproject_toml(context: dict[str, Any], provider_details: Provid
     context["EXTRAS_REQUIREMENTS"] = "\n".join(optional_dependencies)
     context["DEPENDENCY_GROUPS"] = "\n".join(dependency_groups)
 
+    # Most providers require the same python versions, but some may have exclusions
+    requires_python_version = "~=3.9"
```

Comment:
NIT: we coud use `DEFAULT_PYTHON_MAJOR_MINOR_VERSION` from global_constants.py here. This way we will not have to change it when we bump to 3.10.

Reply:
Yupp, had this thought as I went to bed last night, I didn't like this magic string here in the middle of the code. I'll update this